commands.js

```diff
@@ -1892,6 +1892,28 @@ exports.commands = {
 	},
 	hidetexthelp: ["/hidetext [username] - Removes a locked or banned user's messages from chat (includes users banned from the room). Requires: % (global only), @ * # & ~"],
 
+	unlogtext: function (target, room, user) {
+		if (!target) return this.parse('/help unlogtext');
+		this.splitTarget(target);
+		let targetUser = this.targetUser;
+		let name = this.targetUserName;
+		if (!user.can("rangeban")) {
+			this.errorReply("/unlogtext - Access denied.");
+			return false;
+		}
+		if (!targetUser) return this.errorReply("User '" + name + "' not found.");
+		if (!(targetUser.locked || (room.bannedUsers[toId(name)] && room.bannedIps[targetUser.latestIp]) || user.can('rangeban'))) return this.errorReply("User '" + name + "' is not banned from this room or locked.");
```

Comment:
This is the "is the user banned" check. It's the same as the one in the hidetext command.